src/App.tsx

```diff
@@ -96,18 +97,18 @@ const App: React.FC = () => {
             data={[
               {
                 name: 'Home',
-                link: '/'
+                url: '/'
```

Comment:
Use `routes.home` reference here too, so it's consistent throughout your application.